mappings/servers/fara/metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "fara",
+    "name": "FaraMC",
+    "description": "The next generation of Minecraft Minigames.",
+    "addresses": [
+        "faramc.uk"
+    ],
+    "primaryAddress": "faramc.uk",
+    "minecraftVersions": [
+        "1.8.*",
+        "1.20.*"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#800080",
+    "secondaryColor": "#6a5acd",
+    "primaryLanguage": "en",
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "EU",
+    "regions": [
+        "EU"
+    ],
+    "gameTypes": [
+        "PvP", "Minigames"
+    ],
+    "website": "https://faramc.uk",
+    "store": "https://store.faramc.uk",
+    "compliance": {
+        "privacyPolicy": "https://faramc.uk/privacy",
+        "termsOfService": "https://faramc.uk/terms",
+        "rules": "https://faramc.uk/rules"
+    },
+    "socials": {
+        "twitter": "FaraMCX",
+        "youtube": "FaraMCYT",
+        "discord": "https://www.discord.com/invite/2H5Z66StvM"
```

Comment:
Use a `discord.gg` invite link instead

So, `discord.gg/2H5Z66StvM`

You don't need to include `https://` or `www`